--- readline-6.3/support/shlib-install.0006~	2014-03-03 03:00:05.487327542 +0100
+++ readline-6.3/support/shlib-install	2014-03-03 03:00:49.078646055 +0100
@@ -73,7 +73,7 @@ fi
 case "$host_os" in
 hpux*|darwin*|macosx*|linux*|solaris2*)
 	if [ -z "$uninstall" ]; then
-		chmod 555 ${INSTALLDIR}/${LIBNAME}
+		chmod 755 ${INSTALLDIR}/${LIBNAME}
 	fi ;;
 cygwin*|mingw*)
 	IMPLIBNAME=`echo ${LIBNAME} \
